test/connect.test.ts

```diff
@@ -404,25 +404,23 @@ describe('shouldThrottleRequest', () => {
     } as IncomingMessage)).toEqual(false);
   });
 
-  it('maxIpRateHostViolation flags host violations as badIp if IP rate is exceeded', () => {
+  it('maxIpRateHostViolation flags host violations as hostViolation only if IP rate is exceeded', () => {
```

Comment:
looks like there is a missing test to also verify the `badIp` threshold?

Reply:
We will no longer return `badIp` for host violations - this test is verifying that `hostViolation` is returned only after the overridden IP limit is exceeded. There was an existing unit test above this one that verifies the normal `hostViolation` case (with no `maxIpRateHostViolation` specified)